lib/kubernetes-deploy/kubernetes_resource/cloudsql.rb

```diff
@@ -6,17 +6,17 @@ class Cloudsql < KubernetesResource
     def sync
       _, _err, st = kubectl.run("get", type, @name)
       @found = st.success?
-      @status = if cloudsql_proxy_deployment_exists? && mysql_service_exists?
+      @deployment_exists = cloudsql_proxy_deployment_exists?
+      @service_exists = mysql_service_exists?
+      @status = if @deployment_exists && @service_exists
```

Comment:
These changes aren't strictly related to this PR. I just noticed that these resources were effectively re-syncing every time we called `deploy_succeeded?`